Overview.src.html

```diff
@@ -3355,120 +3370,81 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 due course.
 <!-- https://lists.w3.org/Archives/Public/public-whatwg-archive/2014Jun/thread.html#msg72 -->
 
-<p>Objects implementing the <code>Body</code> mixin gain an associated
-<dfn title=concept-Body-body>body</dfn> (a byte stream), a
-<dfn title=concept-Body-used-flag>used flag</dfn> (initially unset), and a
+<p>Objects implementing the <code>Body</code> mixin gain a
 <dfn title=concept-Body-MIME-type>MIME type</dfn> (initially the empty byte sequence).
 
+<p>Objects implementing the <code>Body</code> mixin must define an associated
+<dfn title=concept-Body-disturbed>disturbed</dfn> predicate and a
+<dfn title=concept-Body-consume-body>consume body</dfn> algorithm.
+
 <p>The <dfn title=dom-Body-bodyUsed><code>bodyUsed</code></dfn> attribute's getter must
-return true if the <span title=concept-Body-used-flag>used flag</span> is set, and false
-otherwise.
+return true if <span title="concept-Body-disturbed">disturbed</span>, and false otherwise.
 
 <p>Objects implementing the <code>Body</code> mixin also have an associated
-<dfn title=concept-Body-consume-body>consume body</dfn> algorithm, which given a
-<var title>type</var>, runs these steps:
+<dfn title=concept-Body-package-data>package data</dfn> algorithm, which given
+<var>bytes</var>, a <var>type</var> and a <var>MIME type</var>, switches on <var>type</var>, and
+runs the associated steps:
 
-<ol>
- <li><p>Let <var title>p</var> be a new promise.
+<dl class=switch>
+ <dt><i title>ArrayBuffer</i>
+ <dd><p>Return an <code>ArrayBuffer</code> whose contents are <var>bytes</var>.
```

Comment:
Are you OK with https://github.com/yutakahirano/fetch/commit/bf979b16cf394dc0b84ef2716dbce4b264800d8a?